examples/ui-demo/src/components/shared/mint-card/MintCard.tsx

```diff
@@ -123,15 +123,15 @@ export const MintCard = () => {
             <ValueProps status={status} />
           </div>
           <div
-            className={`p-6 xl:px-10 xl:py-12 h-full bg-bg-surface-default  xl:bg-bg-surface-inset md:min-w-96`}
+            className={`p-6 xl:px-10 xl:py-12 h-full bg-bg-surface-default  xl:bg-bg-surface-inset`}
           >
             <NFT
               nftTransfered={nftTransfered}
               transactionUrl={transactionUrl}
               status={status}
             />
             <MintCardActionButtons
-              className="hidden xl:block"
+              className="hidden xl:block w-[208px] m-auto"
```

Comment:
any reason for the `w` of this to be controlled by `flex-1` and use padding / break points to control the spacing and layout?

Reply:
they want it to match the width of the mint image which isn't a direct sibling